### PR TITLE
Use Go .gitignore

### PR DESCRIPTION
**Pull request by csscottlogic**
From https://github.com/github/gitignore

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,254 +1,30 @@
...
-# Output dir (build)
-build/output/
-
-# Nuget stuff (can be install with build.sh)
-.nuget/
-
-# Ubuntu trash folder
-.Trash-*
-
-# Swap files
```
Comment (dwybourn) [inline .gitignore]: I'd keep the swap files in, as they're created by vim.

File: .gitignore [hunk 2]
```diff
@@ -1,254 +1,30 @@
...
-front/npm-debug.log.*
-
-# Output dir (build)
-build/output/
-
-# Nuget stuff (can be install with build.sh)
-.nuget/
-
-# Ubuntu trash folder
-.Trash-*
```
Comment (jebbinSCL) [inline .gitignore]: I'd keep the .Trash folder as well, as its not clear when it might be created, that way it doesn't pop up in the repo again.

**Comment by csscottlogic** [comment]
I've added back in the Ubuntu trash folder and the Vim swap files

**Comment by jebbinSCL** [comment]
:shipit:

**Comment by dwybourn** [comment]
:shipit:

**Pull request merged**